File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/mr/knowledgetype/d/Attribute.java
package iih.ci.mr.knowledgetype.d;

import xap.sys.appfw.orm.desc.ent.dataobject.DmEnumDesc;

public interface Attribute {

    @DmEnumDesc(name="个人",description="个人")
	public static final Integer PERSONAL=1; //个人
    @DmEnumDesc(name="科室",description="科室")
	public static final Integer DEPT=2; //科室
}	

File: client/iih_stub/iih.ci_stub/iih.ci.mr/knowledgetype/d/attribute.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.ComponentModel;

namespace iih.ci.mr.knowledgetype.d
{
   public enum Attribute {
	/// <summary>
	/// 个人
	/// </summary>
    [Description("个人")]
    PERSONAL=1,
	/// <summary>
	/// 科室
	/// </summary>
    [Description("科室")]
    DEPT=2,
   }
}
